vector-search/vector-search-quickstart.ipynb

```diff
@@ -0,0 +1,703 @@
+{
```

Comment:
<p>For this notebook, I think it would be useful to include how to get the data from BQ and generate the embeddings using Vertex AI Embeddings API.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
Added a reference to the longer notebook with everything (BQ, Emb and VS). The PM asked me to shorten this Notebook as possible as a quick start so we skipped the non-VS related part.